src/styles/variables.scss

```diff
@@ -0,0 +1,19 @@
+@use "./variables/color-darkmode.scss" as color-dark;
```

Comment:
Probably better would be to rename file to `darkmode-colors.scss` and this alias to `darkmode-colors` or `dark-colors` or even maybe to `dark-theme`